gluetool/tool.py

```diff
@@ -440,7 +440,7 @@ def run_pipeline(self):
                 return failure, destroy_failure
 
             if failure and isinstance(failure.exception, GlueRetryError):
-                Glue.error(failure.exception)
+                Glue.error(str(failure.exception))
```

Comment:
@happz why is this conversation needed now?

Reply:
Because `Glue.error` accepts `str` as its first argument. An implicit conversion would happen, Python would call `str(failure.exception)` for us, but it wouldn't pass the type check since we were passing `Exception` where `str` is required. It wasn't a problem until now because `Glue.error` didn't have usable type annotation, that comes with this patch.